torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -196,6 +196,16 @@ void TensorView::setMaxProducer(unsigned int pos) {
   max_producer_pos_ = pos;
 }
 
+void TensorView::decreaseComputeAtPositionTo(unsigned int this_pos) {
```

Comment:
Let's not add new computeAt methods on TensorViews. Just call setComputeAt from compute_at

Reply:
I added these because original `setComputeAt` was only meant to increase the positions. Added an option on `setComputeAt` to support decrease and defaulted to false.